test/augmentation/test_augmentation.py

```diff
@@ -166,52 +159,12 @@ def _test_smoke_call_implementation(self, params):
         test_input = torch.ones(batch_shape, device=self.device, dtype=self.dtype)
-        augmentation = self._create_augmentation_from_params(**params, return_transform=False)
+        augmentation = self._create_augmentation_from_params(**params)
         generated_params = augmentation.forward_parameters(batch_shape)
-        test_transform = torch.rand(expected_transformation_shape, device=self.device, dtype=self.dtype)
 
         output = augmentation(test_input, params=generated_params)
         assert output.shape[0] == batch_shape[0]
-
-        output, transformation = augmentation(test_input, params=generated_params, return_transform=True)
-        assert output.shape[0] == batch_shape[0]
-        assert transformation.shape == expected_transformation_shape
-
-        output, final_transformation = augmentation(
-            (test_input, test_transform), params=generated_params, return_transform=True
-        )
-        assert output.shape[0] == batch_shape[0]
-        assert final_transformation.shape == expected_transformation_shape
-        assert_close(final_transformation, transformation @ test_transform, atol=1e-4, rtol=1e-4)
-
-        output, transformation = augmentation((test_input, test_transform), params=generated_params)
-        assert output.shape[0] == batch_shape[0]
-        assert transformation.shape == expected_transformation_shape
-        assert (transformation == test_transform).all()
-
-    def _test_smoke_return_transform_implementation(self, params):
-        batch_shape = (4, 3, 5, 6)
-        expected_transformation_shape = torch.Size((batch_shape[0], 3, 3))
-        test_input = torch.ones(batch_shape, device=self.device, dtype=self.dtype)
-        augmentation = self._create_augmentation_from_params(**params, return_transform=True)
-        generated_params = augmentation.forward_parameters(batch_shape)
-        test_transform = torch.rand(expected_transformation_shape, device=self.device, dtype=self.dtype)
-
-        output, transformation = augmentation(test_input, params=generated_params)
-        assert output.shape[0] == batch_shape[0]
-        assert transformation.shape == expected_transformation_shape
-
-        output, final_transformation = augmentation((test_input, test_transform), params=generated_params)
-        assert output.shape[0] == batch_shape[0]
-        assert final_transformation.shape == expected_transformation_shape
-        assert_close(final_transformation, transformation @ test_transform, atol=1e-4, rtol=1e-4)
-
-        output, final_transformation = augmentation(
-            (test_input, test_transform), params=generated_params, return_transform=True
-        )
-        assert output.shape[0] == batch_shape[0]
-        assert final_transformation.shape == expected_transformation_shape
-        assert_close(final_transformation, transformation @ test_transform, atol=1e-4, rtol=1e-4)
+        assert augmentation._transform_matrix.shape == expected_transformation_shape
```

Comment:
```suggestion
        assert augmentation.transform_matrix.shape == expected_transformation_shape
```
Better to show with property